index.js

```diff
@@ -22,69 +23,85 @@ function getRandomNumber(min, max) {
   return Math.floor(Math.random() * (max - min)) + min;
 }
 
+const messageList = document.querySelector(".messy");
+const errorMessage = document.createElement("p");
+
+hideAllMessages();
+
 function checkGuess() {
   // Get value from guess input element
   const guess = parseInt(guessInput.value, 10);
   attempts = attempts + 1;
 
-  hideAllMessages();
-
   if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
+    numberOfGuessesMessage.style.display = "";
     numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
 
-    correctMessage.style.display = '';
+    correctMessage.style.display = "";
 
     submitButton.disabled = true;
     guessInput.disabled = true;
   }
 
+  if (guess > 100) {
+    moreThanHundredMessage.style.display = "";
+  }
+
+  if (guess < 0) {
+    lessThanZeroMessage.style.display = "";
+  }
+
   if (guess !== targetNumber) {
     if (guess < targetNumber) {
-      tooLowMessage.style.display = '';
+      tooLowMessage.style.display = "";
     } else {
-      tooLowMessage.style.display = '';
+      tooHighMessage.style.display = "";
     }
 
     const remainingAttempts = maxNumberOfAttempts - attempts;
 
-    numberOfGuessesMessage.style.display = '';
-    numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
+    console.log(numberOfGuessesMessage);
+    numberOfGuessesMessage.style.display = "";
+    if (remainingAttempts > 1) {
+      numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
+    } else {
+      numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guess remaining`;
+    }
   }
 
-  if (attempts ==== maxNumberOfAttempts) {
+  if (attempts === maxNumberOfAttempts) {
     submitButton.disabled = true;
     guessInput.disabled = true;
   }
 
-  guessInput.value = '';
+  guessInput.value = "";
```

Comment:
You're setting your input value to a string when your input is expecting a number. If you click your button without inputting a number, you'll see that your guess is `NaN` which means `Not a Number`. This is and can be the source of bugs. Any easy way to avoid this is just to keep your types the same here by setting `guessInput.value = 0` instead of an empty string.